crates/solvers/example.baseline.toml

```diff
@@ -1,4 +1,5 @@
 chain-id = "1"
-weth = "0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2"
+# Alternatively, you can manually specify a WETH contract address:
```

Comment:
nit: Could you make the comment refer to the `chain-id` parameter? This makes the dependency more explicit and less error prone if somebody where to insert a new argument between `chain-id` and `weth`.